angular/projects/spark-angular/src/lib/components/sprk-masthead/components/sprk-masthead-accordion/sprk-masthead-accordion.component.ts

```diff
@@ -3,12 +3,16 @@ import { Component, Input } from '@angular/core';
 @Component({
   selector: 'sprk-masthead-accordion',
   template: `
-    <nav role="navigation" [attr.data-id]="idString">
+    <nav
+      role="navigation"
+      [attr.data-id]="idString"
+      [attr.aria-label]="ariaLabel"
```

Comment:
I added a new `aria-label` input since this is a nav element and might need to have that set.